benchexec/tools/coveriteam-verifier-validator.py

```diff
@@ -0,0 +1,32 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.coveriteam as coveriteam
+
+
+class Tool(coveriteam.Tool):
+    """
+    Tool info for a verifier or a validator based on
+    CoVeriTeam: a Configurable Software-Verification Platform.
+    URL: https://gitlab.com/sosy-lab/software/coveriteam.
+    """
+
+    def name(self):
+        return "Verifier (or Validator) Based on CoVeriTeam"
+
+    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
+        """
+        Prepare command for the example coveriteam program for a verifier.
```

Comment:
The class description claims that this tool-info module is also for validators. Where is the command line for them?

Reply:
This function produces command for both.
as with any parameter which is not program or property: it is in the options.

Reply:
Then the comment needs to be updated, otherwise it is highly confusing. Also, why does it mention an example program?